addons/website/static/src/scss/bootstrap_overridden.scss

```diff
@@ -250,8 +250,8 @@ $form-check-input-border: ($input-border-width or 1px) solid $input-border-color
 $form-range-track-bg: $light !default;
 
 // Badge
-$badge-padding-y: 0.25em !default;
-$badge-padding-x: 0.4em !default;
+$badge-padding-y: .33em !default;
+$badge-padding-x: .66em !default;
```

Comment:
Can't you just remove those rules?
BS default ones seems to be 
```scss
$badge-padding-y:                   .35em !default;
$badge-padding-x:                   .65em !default;
```
Might as well just use those, they are the exact same almost?
(Those are overriden in bootstrap_overridden.scss from web but it's not included in the frontend)

Reply:
Good point.

We tried some different values and ended to select this one which was the best one. But I did not realise it was so close to BS default. So I think it makes sense to do as you are suggesting.